File: 0001-two-sum/0001-two-sum.cpp
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> index;
        for(unsigned int i = 0; i < nums.size()-1; i++){
            for(unsigned int j = i+1; j < nums.size(); j++){
                if(nums[i]+nums[j] == target){
                    index.push_back(i);
                    index.push_back(j);
                    return index;
                }
            }
        }
        return index;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans = []
        mem = {}
        for i in range(0,len(nums)):
            if (target-nums[i] in mem.keys()):
                ans.append(mem[target-nums[i]])
                ans.append(i)
                return ans
            else:
                mem[nums[i]] = i
